.github/workflows/helm-vib-lint.yaml

```
@@ -0,0 +1,26 @@
+name: Lint Helm Chart
+on:
+  workflow_dispatch:
+  pull_request_target:
+    branches:
+      - main
+      - bitnami-labs:main
+    paths:
+       - 'helm/**'
```

```suggestion
  pull_request_target:
    branches:
       - main
       - bitnami-labs:main
    paths:
       - 'helm/**'
```

Alignment

---

You are right, but the misalignment is on the paths I think.

```suggestion
  pull_request_target:
    branches:
      - main
      - bitnami-labs:main
    paths:
      - 'helm/**'
```